npm/private/yaml.bzl

```diff
@@ -55,6 +55,10 @@ def parse(yaml):
             msg = "Unknown state {}".format(state["id"])
             fail(msg)
 
+    if len(starlark["errors"]) > 0:
+        print("YAML parse error(s): ", starlark["errors"])
+        return None
```

Comment:
Do we handle the `None` case where needed in rules_js?

Reply:
We already return `None` in a few cases and the pnpm-lock logic treats it as an empty lockfile. So now an invalid lockfile outputs an error and treats it as empty, instead of crashing.

I'm not sure if that's the best solution but it's better then crashing? 😅